ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -1703,11 +1697,9 @@ tests:
     cluster_profile: gcp-qe
     env:
       FIPS_ENABLED: "true"
-      TEST_FILTERS: ~ChkUpgrade&;~ConnectedOnly&;~MicroShiftOnly&
```

Comment:
@wangke19 
should add 
```yaml
TEST_FILTERS_DISASTERRECOVERY: ~ChkUpgrade&;NonPreRelease&;~ ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
because it is disconnected env.

the default filter is for connected env.

Reply:
Updated.